bot/exts/utilities/bookmark.py

```diff
@@ -16,14 +18,51 @@
 # Number of seconds to wait for other users to bookmark the same message
 TIMEOUT = 120
 BOOKMARK_EMOJI = "📌"
+MESSAGE_NOT_FOUND_ERROR = (
+    "You must either provide a valid message to bookmark, or reply to one."
+    "\n\nThe lookup strategy for a message is as follows (in order):"
+    "\n1. Lookup by '{channel ID}-{message ID}' (retrieved by shift-clicking on 'Copy ID')"
+    "\n2. Lookup by message ID (the message **must** have been sent after the bot last started)"
```

Comment:
In the same channel right?

Reply:
Nope, that's not how it works in 2.0 now.